src/Maker/MakeDockerDatabase.php

```diff
@@ -0,0 +1,154 @@
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command): void
+    {
+        $io->section('- Docker Compose Setup-');
+
+        $composeFileContents = '';
+        $statusMessage = 'Existing docker-compose.yaml not found: a new one will be generated!';
+
+        if ($this->fileManager->fileExists($this->composeFilePath)) {
+            $composeFileContents = $this->fileManager->getFileContents($this->composeFilePath);
+
+            $statusMessage = 'We found your existing docker-compose.yaml: Let\'s update it!';
+        }
+
+        $io->text($statusMessage);
+
+        $this->composeFileManipulator = new ComposeFileManipulator($composeFileContents);
+
+        $io->newLine();
+
+        $this->databaseChoice = strtolower($io->choice(
+            'Which database service will you be creating?',
+            ['MySQL', 'MariaDB', 'Postgres']
+        ));
+
+        $io->text([sprintf(
+            'For a list of supported versions, check out https://hub.docker.com/_/%s',
+            $this->databaseChoice
+        )]);
+
+        $this->serviceVersion = $io->ask('What version would you like to use?', DockerDatabaseServices::getSuggestedServiceVersion($this->databaseChoice));
+
+        if ($this->composeFileManipulator->serviceExists($this->serviceName)) {
+            $io->comment(sprintf('A <fg=yellow>"%s"</> service is already defined.', $this->serviceName));
+            $io->newLine();
+
+            $envTest[] = 'If you are using the Symfony Binary, it will expose the env_var associated with this service.';
+            $envTest[] = 'If you name the service \'database\', the binary will look for DATABASE_URL';
+
+            $io->text($envTest);
+
+            $this->serviceName = $io->ask(sprintf('What name should we call the new %s service? e.g. database', $this->serviceName));
+        }
+    }
+
+    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator): void
+    {
+        $io->newLine();
+
+        $service = DockerDatabaseServices::getDatabaseSkeleton($this->databaseChoice, $this->serviceVersion);
+
+        $this->composeFileManipulator->addDockerService($this->serviceName, $service);
+
+        $generator->dumpFile($this->composeFilePath, Yaml::dump($this->composeFileManipulator->getComposeData(), 20));
+        $generator->writeChanges();
+
+        $this->writeSuccessMessage($io);
+
+        $io->text(sprintf('The new <fg=yellow>"%s"</> service is now ready.', $this->serviceName));
+        $io->newLine();
+
+        $closing[] = 'Next:';
+        $closing[] = sprintf('Run <fg=yellow>docker-compose up -d %s</> to start your database container or <fg=yellow>docker-compose up -d</> to start all of them.', $this->serviceName);
+        $closing[] = 'If you are using the Symfony Binary, it will detect the new service automatically.';
+        $closing[] = 'Run <fg=yellow>symfony var:export --multiline</> to see the environment variables exposed for Docker.';
```

Comment:
```suggestion
        $closing[] = 'Run <fg=yellow>symfony var:export --multiline</> to see the environment variables the binary is exposing.';
```

... edge case... and I'm not even sure *what* the edge case would be... but this prints all env variables, not just the ones from docker.